components/brave_wallet_ui/components/desktop/views/crypto/index.tsx

```diff
@@ -1,7 +1,18 @@
 import * as React from 'react'
 
 import { StyledWrapper } from './style'
-import { TopTabNavTypes, AppObjectType, AppsListType } from '../../../../constants/types'
+import {
+  TopTabNavTypes,
+  AppObjectType,
+  AppsListType,
+  ChartTimelineType,
+  PriceDataObjectType,
+  AssetOptionType,
+  UserAssetOptionType,
+  RPCTransactionType,
+  AssetPriceReturnInfo,
+  WalletAccountType
+} from '../../../../constants/types'
```

Comment:
nit: Since these are just typescript types then there's no size issue just `import * as Types from '../../../../constants/types'` if this is annoying overhead for you to import each item individually everywhere!